api/fixtures/example.go

```diff
@@ -13,23 +13,29 @@ import (
 )
 
 type ExampleResources struct {
```

Comment:
Just a note, no action necessary here (unless you want), this struct and approach to building the resources was just a quick part of the original prototypes that was never reconsidered. If there's a factoring that makes it easier to compose stuff based on platform variables, I'd be happy to see it refactored.

Reply:
Good to know. I think it works reasonably well so far. It was already structured pretty nice. I could move platform specific parts to their own files, but maybe let's wait for additional providers.